config/coq_config.mli

```diff
@@ -28,11 +28,11 @@ val ocamllex : string
 val camlbin : string    (* base directory of OCaml binaries *)
 val camllib : string    (* for Dynlink *)
 
-val camlp4 : string     (* exact name of camlp4: either "camlp4" ou "camlp5" *)
-val camlp4o : string    (* name of the camlp4o/camlp5o executable *)
-val camlp4bin : string  (* base directory for Camlp4/5 binaries *)
-val camlp4lib : string  (* where is the library of Camlp4 *)
-val camlp4compat : string (* compatibility argument to camlp4/5 *)
+val camlp5 : string     (* exact name of camlp5, i.e. "camlp5" *)
```

Comment:
This variable and the logic associated to it can probably be removed since it only served to choose between camlp4 and camlp5.

Reply:
AFAICT this was not related to choice but to install location in pre-ocamlfind times.

Reply:
The two suggested values in the original code are "camlp5" and "camlp4".

Reply:
Oh indeed I misread the comment as affecting all the 3 lines, you are right.

Reply:
I get confused sometimes by the github context, I wish there was a way to mark the exact span of one is commenting.

Reply:
Would it make your life easier if I submitted these additional changes as a separate commit, with the idea that you could squash 2 commits into one before merging?  That way you'd only have to review the additional changes (and not re-read the earlier ones).

Reply:
Both solutions are fine to me.